client/app/src/Command/ProcessLayCSVCommand.php

```diff
@@ -54,32 +54,42 @@ protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $bucket = $this->params->get('s3_sirius_bucket');
         $layReportFile = $this->params->get('lay_report_csv_filename');
+        $fileLocation = sprintf('/tmp/%s', $layReportFile);
 
         try {
             $this->s3->getObject([
```

Comment:
If we have the command on the api side now, is this command still required on the client side?

Reply:
Yes, as it's part of the manual process. Which is one of the AC's for the ticket.